snippets/perl/README.md

```diff
@@ -0,0 +1,28 @@
+### NetBackup API Code Samples for Perl
```

Comment:
Could you please add similar README.md for other languages' snippets directories as well?

Reply:
done